app/stac_api/views.py

```diff
@@ -274,6 +300,11 @@ def put(self, request, *args, **kwargs):
     def patch(self, request, *args, **kwargs):
         return self.partial_update(request, *args, **kwargs)
 
+    # Here the etag is only added to support pre-conditional If-Match and If-Not-Match
+    # @etag(get_collection_etag)
+    # def delete(self, request, *args, **kwargs):
+    #     return self.destroy(request, *args, **kwargs)
+
```

Comment:
Intentionally commented out?

Reply:
Yes the implementation of the collection delete is not complete and there is a ticket to do that, see https://jira.swisstopo.ch/browse/BGDIINF_SB-1742. I've added it here to be able to do some E2E test allowing me to clean up the DB after the E2E test.